modules/coreos-layering-configuring-on-modifying.adoc

```diff
@@ -0,0 +1,125 @@
+
+After you make the change, the MCO reboots the nodes associated with the specified machine config pool.
+
+include::snippets//coreos-layering-configuring-on-pause.adoc[]
+
+.Prerequisites
+
+* You have opted-in to on-cluster layering by creating a `MachineOSConfig` object.
+
+.Procedures
+
+* Modify an object to update the associated custom layered image:
+
+.. Optional: Edit the `MachineOSConfig` object to modify the custom layered image. The following example adds the rngd daemon to nodes that already have the tree package installed by using a custom layered image.
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1alpha1
+kind: MachineOSConfig
+metadata:
+  name: layered-alpha1
+spec:
+  machineConfigPool:
+    name: layered
+  buildInputs:
+    containerFile: 
+    - containerfileArch: noarch
+      content: |- <1>
+        FROM configs AS final
+  
+        RUN rpm-ostree install rng-tools && \
+            systemctl enable rngd && \
+            rpm-ostree cleanup -m && \
+            ostree container commit
+  
+        RUN rpm-ostree install tree && \
+            ostree container commit        
+    imageBuilder: 
+      imageBuilderType: PodImageBuilder
+    baseImagePullSecret: 
+      name: global-pull-secret-copy <2>
+    renderedImagePushspec: image-registry.openshift-image-registry.svc:5000/openshift-machine-config-operator/os-images:latest <3>
+    renderedImagePushSecret:  <4>
+      name: new-secret-name
+  buildOutputs: 
+    currentImagePullSecret:
+      name: new-secret-name <5>
+----
+<1> Optional: Modify the containerFile, for example to add or remove packages. 
+<2> Optional: Update the secret needed to pull pull the base operating system image from the registry.
+<3> Optional: Modify the image registry to push the newly-built custom layered image to.
+<4> Optional: Update the secret needed to push the base operating system image to the registry.
+<5> Optional: Update the secret needed to pull the base operating system image from the registry.
++
+When you save the changes, the MCO drains, cordons, and reboots the nodes. After the reboot, the node uses the cluster base {op-system-first} image.
+
+.Verification
+
+. Verify that the new `MachineOSBuild` object was created by using the following command:
++
+[source,terminal]
+----
+$ oc get machineosbuild
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                       PREPARED   BUILDING   SUCCEEDED   INTERRUPTED   FAILED
+layered-a5457b883f5239cdcb71b57e1a30b6ef   False      False      True        False         False
+layered-f91f0f5593dd337d89bf4d38c877590b   False      True       False       False         False <1>
+----
+<1> The value `True` in the `BUILDING` column indicates that the `MachineOSBuild` object is building. When the field becomes `False`, the new custom layered image has rolled out.
```

Comment:
```suggestion
<1> The value `True` in the `BUILDING` column indicates that the `MachineOSBuild` object is building. When the field becomes `False` and the `SUCCEEDED` column becomes `True`, the new custom layered image has been built successfully and is being rolled out onto the nodes in the pool.
```

Maybe, after this we can mention how to observe the rollout by checking the status of the MCP.